### PR TITLE
Add new healthcheck for Local Links Manager

### PR DESCRIPTION
**Pull request by callumknights**
Added so continuous deployment can be enabled for
Local Links Manager.

Trello: https://trello.com/c/uAx6eKAQ/2276-activate-continuous-deployment-for-local-links-manager

## Testing

**You should manually test your PR before merging.**

This app doesn't have CI setup, and it would be misleading to do so:
the behaviour of the tests changes depending on the environment they
are run in. You should manually test in applicable environments,
until you are confident your change is not a breaking one.

Example steps to test in Integration:

- Click "Configure" for the [Smokey job][]
- Change the "Branch specifier" to the name of your branch
- Run a new build of the Smokey project and check the results

## Deployment

**You need to manually deploy your PR after merging.**

Steps to deploy a change:

- Run the [Smokey deploy job][] to deploy the [continuous Smokey loop][]
- Repeat this in Staging and Production

> The manual [Smokey job][] will pick up changes on `master` automatically. You only need to do a manual deployment for the [continuous Smokey loop][].

[Smokey job]: https://deploy.integration.publishing.service.gov.uk/job/Smokey/
[continuous Smokey loop]: https://github.com/alphagov/govuk-puppet/blob/master/modules/monitoring/templates/smokey-loop.conf
[Smokey deploy job]: https://deploy.integration.publishing.service.gov.uk/job/Smokey_Deploy/

### CODE REVIEW

**Review by thomasleese — COMMENTED** [review]
I think we need to add `@app-local-links-manager` to this: https://github.com/alphagov/smokey/blob/1efb2db3c2867ee3a27a8375279aebbc653eb75a/features/publishing_tools.feature#L47-L53

File: features/local_links_manager.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@app-local-links-manager
+Feature: Local Links Manager
+
+  @local_network
+  Scenario: Healthcheck
+    Given: I am testing "local-links-manager" internally
+    When: I request "/healthcheck"
+    Then: Json is returned
+    And: I should see ""status":"ok""
```
Comment (thomasleese) [inline features/local_links_manager.feature:9]: I think usually the `:` wouldn't be on these lines (I'm not sure if it works or not). For example: https://github.com/alphagov/smokey/blob/1efb2db3c2867ee3a27a8375279aebbc653eb75a/features/service_manual_publisher.feature#L6-L8

File: features/local_links_manager.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@app-local-links-manager
+Feature: Local Links Manager
+
+  @local_network
+  Scenario: Healthcheck
+    Given I am testing "local-links-manager" internally
+    When I request "/healthcheck"
+    Then Json is returned
```
Comment (thomasleese) [inline features/local_links_manager.feature:8]:
```suggestion
    Then JSON is returned
```

Not sure if this works either way, but [JSON would normally be all capitalised](https://github.com/alphagov/smokey/search?q=JSON+is+returned).

**Review by thomasleese — COMMENTED** [review]
Looks good, just got one comment!

File: features/local_links_manager.feature (hunk 2 above)
Comment (callumknights) [reply features/local_links_manager.feature:8]: 👍

**Review by MuriloDalRi — APPROVED** [review]

**Comment by MuriloDalRi** [comment]
What @thomasleese said

**Pull request merged**